### ISSUE TITLE
Rest-api-example failed to start the SUT

### ISSUE THREAD

**Issue by Minju-Kan** [EMResearch/EvoMaster#347]
Hi, I try the project named rest-api-example, using evomaster1.2.1.jar. 
Then there is an exception result ：

```
This is likely a bug in EvoMaster. Please copy&paste the following stacktrace, and create a new issue on ?[34mhttps://github.com/EMResearch/EvoMaster/issues?[0m?[0m
java.lang.IllegalStateException: Failed to start the SUT
        at org.evomaster.core.Main$Companion.init(Main.kt:233)
        at org.evomaster.core.Main$Companion.initAndRun(Main.kt:140)
        at org.evomaster.core.Main$Companion.main(Main.kt:73)
        at org.evomaster.core.Main.main(Main.kt)
```
Is that a problem because I'm not using evomaster1.0.1.jar (Your video using this version) ?

**Comment by arcuri82** [comment]
Hi,
 have you started the driver before running `evomaster1.2.1.jar`? (currently 1.2.1 is giving wrong error message when driver is not up and running).

**Comment by arcuri82** [comment]
this should be now fixed in the new `1.3.0` release
